packages/config/config/devices/0x000c/hs-wx300.json

```diff
@@ -0,0 +1,207 @@
+{
+	"manufacturer": "HomeSeer Technologies",
+	"manufacturerId": "0x000c",
+	"label": "HS-WX300",
+	"description": "Wall Dimmer & Switch",
+	"devices": [
+		{
+			"productType": "0x4447",
+			"productId": "0x4037"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Companion Sync",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "3",
+			"label": "Bottom LED Operation",
+			"$import": "~/templates/master_template.json#led_indicator_two_options"
+		},
+		{
+			"#": "4",
+			"label": "Orientation",
+			"description": "Sets paddle load orientation",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Top of Paddle turns load ON",
+					"value": 0
+				},
+				{
+					"label": "Bottom of Paddle turns load ON",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "5",
+			"label": "Dimming Threshold",
+			"description": "Sets the lowest dimming threshold",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 14,
+			"defaultValue": 1
+		},
+		{
+			"#": "6",
+			"label": "Central Scene Control",
+			"description": "Enables/Disables Central Scene and multiple tap functions",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Central Scene Enabled, controls load with delay. Enables Multi-tap and",
+					"value": 0
+				},
+				{
+					"label": "Central Scene Disabled, controls load instantly. Disables multi-tap, c",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "11",
+			"label": "Remote Ramp Rate",
+			"description": "Set dimmer ramp rate for remote control",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 90,
+			"defaultValue": 3
+		},
+		{
+			"#": "12",
+			"label": "Local Ramp Rate",
+			"description": "Set dimmer ramp rate for local control",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 90,
+			"defaultValue": 3
+		},
```

Comment:
Align to https://github.com/zwave-js/node-zwave-js/blob/b23ee76e6bb654ef76580fb5518f9e2e0a14dff6/packages/config/config/devices/0x000c/hs-wd200.json#L26-L43 for consistency please